src/discussions/posts/post-editor/PostEditor.test.jsx

```diff
@@ -110,50 +109,40 @@ describe('PostEditor', () => {
       });
       await executeThunk(fetchCourseTopics(courseId), store.dispatch, store.getState);
     });
-    test(
-      `new post when anonymous posts are ${allowAnonymous ? '' : 'not '}allowed and anonymous posts to peers are ${allowAnonymousToPeers ? '' : 'not '}allowed`,
-      async () => {
-        await renderComponent();
-
-        expect(screen.queryByRole('heading'))
-          .toHaveTextContent('Add a post');
-        expect(screen.queryAllByRole('radio'))
-          .toHaveLength(2);
-        // 2 categories with 4 subcategories each
-        expect(screen.queryAllByText(/category-\d-topic \d/))
-          .toHaveLength(8);
-        // 3 non courseare topics
-        expect(screen.queryAllByText(/ncw-topic \d/))
-          .toHaveLength(3);
-
-        expect(screen.queryByText('cohort', { exact: false }))
-          .not.toBeInTheDocument();
-        if (allowAnonymous) {
-          expect(screen.queryByText('Post anonymously'))
-            .not.toBeInTheDocument();
-        } else {
-          expect(screen.queryByText('Post anonymously'))
-            .not.toBeInTheDocument();
-        }
-        if (allowAnonymousToPeers) {
-          expect(screen.queryByText('Post anonymously to peers'))
-            .toBeInTheDocument();
-        } else {
-          expect(screen.queryByText('Post anonymously to peers'))
-            .not
-            .toBeInTheDocument();
-        }
-      },
-    );
+
+    test(`new post when anonymous posts are ${allowAnonymous ? '' : 'not'} allowed and anonymous posts to peers are ${
+      allowAnonymousToPeers ? '' : 'not'} allowed`, async () => {
+      await renderComponent();
+
+      expect(screen.queryByRole('heading')).toHaveTextContent('Add a post');
+      expect(screen.queryAllByRole('radio')).toHaveLength(2);
+      // 2 categories with 4 subcategories each
+      expect(screen.queryAllByText(/category-\d-topic \d/)).toHaveLength(8);
+      // 3 non courseare topics
+      expect(screen.queryAllByText(/ncw-topic \d/)).toHaveLength(3);
+      expect(screen.queryByText('cohort', { exact: false })).not.toBeInTheDocument();
+
+      if (allowAnonymous) {
+        expect(screen.queryByText('Post anonymously')).not.toBeInTheDocument();
```

Comment:
it should be expect(screen.queryByText('Post anonymously')).toBeInTheDocument();